### ISSUE TITLE
User Dropdown Component Background is Transparent

### ISSUE THREAD

**Issue by nadiemedicejose** [themesberg/flowbite-react#308]
**Describe the bug**
Current background of Avatar Dropdown is transparent.

**To Reproduce**

1. Go to [https://flowbite-react.com/avatar](https://flowbite-react.com/avatar)
2. Copy User dropdown component
3. Use clean react app with base libraries

**Expected behavior**
It should be similar to other dropdown items.

**Screenshots**

![image](https://user-images.githubusercontent.com/44657605/180593150-1ecb886e-bf74-4de3-89c5-617fcf941cf2.png)

**System information:**

- Device: Laptop
- OS: Windows
- Browser: Google Chrome
- Version: 103

**Project information:**

- Tailwind: 3.0.0
- Flowbite: 1.4.7
- Flowbite React: 0.1.2
- Type: CRA

**Additional context**

**Comment by rluders** [comment]
@nadiemedicejose could you try to update flowbite-react to `0.1.3` and see what happens?

**Comment by nadiemedicejose** [comment]
Yes, I updated `flowbite-react` to version `0.1.3` and it's broken too.

> @nadiemedicejose could you try to update flowbite-react to `0.1.3` and see what happens?

**Comment by tulup-conner** [comment]
@nadiemedicejose I just created a Pull Request resolving this issue. I've tested it on [`flowbite-next-starter`](https://github.com/tulupinc/flowbite-next-starter) and some of my other projects using `flowbite-react` so I think it should be good to go for the next release.

**Comment by bamoha** [comment]
Version 0.6.4 and this problem persists.

**Comment by rluders** [comment]
@bamoha update to 0.7.2

**Comment by bamoha** [comment]
![image](https://github.com/themesberg/flowbite-react/assets/16354917/61fdd085-3211-4c59-b41e-642cfdaf4c3d)

It still persists.

**Comment by rluders** [comment]
If someone could create a codepen or something with some example it would be nice.

**Comment by bamoha** [comment]
I found a fix, I added `position: relative` to the main parent div that houses the dropdown, and it fixed it.

**Comment by bamoha** [comment]
It happens on your documentation as well. 

https://www.flowbite-react.com/docs/components/dropdown


<img width="708" alt="image" src="https://github.com/themesberg/flowbite-react/assets/16354917/3f1638d2-fb41-4146-b791-42663875a1f6">

**Comment by bamoha** [comment]
Do we have any updates on this
